cumulus_library/studies/core/core_templates/documentreference.sql.jinja

```diff
@@ -0,0 +1,64 @@
+{% import 'core_utils.jinja' as utils %}
+CREATE TABLE core__documentreference AS
+WITH temp_documentreference AS (
+    SELECT DISTINCT
+        {{- utils.basic_cols(
+                'documentreference',
+                'dr',
+                [
+                    'id',
+                ]
+            ) 
+        }},
+        {{- utils.nullable_cols(
+                'documentreference',
+                'dr',
+                [
+                    'type',
+                    'status',
+                    'docstatus',
+                    'context',                
+                    ('subject', 'reference', 'subject_ref'),
+                    ('context', 'period', 'start', 'author_date')
+                ], 
+                schema
+            ) 
+        }},
+        {{- utils.truncate_date_cols(
+                'documentreference',
+                'dr',
+                [
+                    ('author_date', 'day'),
+                    ('author_date', 'week'),
+                    ('author_date', 'month'),
+                    ('author_date', 'year'),
+                ],
+                schema
+            ) 
+        }},
+    FROM documentreference AS dr
+)
+
+SELECT DISTINCT
+    type_coding.type_row AS doc_type,
+    coalesce(type_coding.type_row.code, 'None') AS doc_type_code,
+    CASE
+        WHEN
+            type_coding.type_row.display IS NOT NULL
+            THEN replace(type_coding.type_row.display, ',', '')
```

Comment:
We can quote our CSV now, yeah? Is this still needed?

If it is still needed, I'd expect this to live in some `get_user_string` macro or something that would be called everywhere we use a display/text value from FHIR.

Reply:
it isn't - and i also realized that this probably shouldn't be here at all, this coding should be in a denormalized table, so some of this may just go away.